cryptoconditions/condition.py

```diff
@@ -1,24 +1,36 @@
-import base58
 import base64
 import re
+from functools import total_ordering
+from itertools import compress
+from urllib.parse import parse_qs
 
 from abc import ABCMeta
 
+from pyasn1.codec.der.decoder import decode as der_decode
+from pyasn1.codec.der.encoder import encode as der_encode
+from pyasn1.codec.native.encoder import encode as nat_encode
+from pyasn1.codec.native.decoder import decode as nat_decode
+
 from cryptoconditions import TypeRegistry
 from cryptoconditions.crypto import base64_add_padding, base64_remove_padding
-from cryptoconditions.lib import Writer, Reader
+from cryptoconditions.exceptions import ParsingError, PrefixError
+
+from cryptoconditions.schemas.condition import Condition as Asn1Condition
 
+CONDITION_URI_SCHEME = 'ni'
 
 # Regex for validating conditions
 # This is a generic, future-proof version of the crypto-condition regular expression.
-CONDITION_REGEX = \
-    r'^cc:([1-9a-f][0-9a-f]{0,3}|0):[1-9a-f][0-9a-f]{0,15}:[a-zA-Z0-9_-]{0,86}:([1-9][0-9]{0,17}|0)$'
+CONDITION_REGEX = r'^ni:\/\/\/sha-256;([a-zA-Z0-9_-]{0,86})\?(.+)$'
+
 # This is a stricter version based on limitations of the current implementation.
 # Specifically, we can't handle bitmasks greater than 32 bits.
-CONDITION_REGEX_STRICT = \
-    r'^cc:([1-9a-f][0-9a-f]{0,3}|0):[1-9a-f][0-9a-f]{0,7}:[a-zA-Z0-9_-]{0,86}:([1-9][0-9]{0,17}|0)$'
+CONDITION_REGEX_STRICT = CONDITION_REGEX
```

Comment:
Is there a reason to have both `CONDITION_REGEX_STRICT` and `CONDITION_REGEX` since they are now the same?